markdown/templates/client_libraries.md

```diff
@@ -57,11 +60,25 @@ client libraries as packages that can be installed via the respective package ma
   <version>14.5.0</version>
 </dependency>
 ```
+<% } else { -%>
```

Comment:
I'd rather use an explicit `elseif` statement here, checking the actual value of `target`. Today, we only support two but who is to say what the future holds?:)